src/utility/utils.ts

```diff
@@ -3,3 +3,5 @@ export const roleTitles = {
   admin: "Admin",
   superadmin: "Super admin",
 };
+
+export const userWaitingForActicityID = "cVkZbjsrXs6YKk3Qli0b";
```

Comment:
The naming here makes it sound as if this is a users id.
I feel like a better name would be `placeholderActivityId`.